src/plugins/enzymes/src/FindEnzymesDialog.h

```diff
@@ -74,6 +74,7 @@ private slots:
     void saveFile(const QString& url);
     void setEnzymesList(const QList<SEnzymeData>& enzymes);
     void updateStatus();
+    int gatherCheckedNamesListString(QString& checkedNamesList);
```

Comment:
nit: add `const`?

Reply:
Nope, this is a hint to avoid extra copying (in out case this is important, because `checkedNamesList` could be very long). If your function is:
```
QString gatherCheckedNamesListString() {
    QString result;
    // do something, result is very large
    return result;
};
```
you call it lake this:
`QString checkedNamesList = gatherCheckedNamesListString(); // copy from return gatherCheckedNamesListString() r-value to checkedNamesList l-value `

To avoid it I pass `checkedNamesList` by reference:
```
int gatherCheckedNamesListString(QString& checkedNamesList) {
    // do something, checkedNamesList is very large
};

QString checkedNamesList;
gatherCheckedNamesListString(checkedNamesList); // no extra copying
```

Actually, this could be handled a bit more modern by using `std::move()`, but I just forgot about it and resolved it in this way.

Reply:
I meant const to the method, not to the parameter. Can the method be const?

Reply:
OK